src/object_store/src/object/s3.rs

```diff
@@ -702,6 +726,24 @@ impl S3ObjectStore {
             }
         }
     }
+
+    #[inline(always)]
+    fn get_retry_strategy() -> Map<Take<ExponentialBackoff>, fn(Duration) -> Duration> {
+        ExponentialBackoff::from_millis(DEFAULT_RETRY_INTERVAL)
+            .max_delay(DEFAULT_RETRY_MAX_DELAY)
+            .take(DEFAULT_RETRY_MAX_ATTEMPTS)
+            .map(jitter)
```

Comment:
FYI, after checking the implementation of `ExponentialBackoff`, with DEFAULT_RETRY_INTERVAL=20ms, DEFAULT_RETRY_MAX_DELAY=5s, DEFAULT_RETRY_MAX_ATTEMPTS=10, I think the current behavior is to generate the following backoff time for each attempt 
```
[20ms * rand(), 400ms * rand(), 5s * rand(), 5s * rand(), 5s * rand(), 5s * rand(), 5s * rand(), 5s * rand(), 5s * rand(), 5s * rand()]
```

I believe when timeout happens, we will most likely hit the `5s * rand()` under the current configs. I am uncertain about whether this is appropriate but it is okay to give it a try.

Reply:
Or maybe we can change `DEFAULT_RETRY_MAX_DELAY = 10s` to make the backoff time more values?
`[20ms * rand(), 400ms * rand(), 8s * rand(), 10s * rand(), 10s * rand(), 10s * rand(), 10s * rand(), 10s * rand(), 10s * rand(), 10s * rand()]`
I think whether peak clipping is possible depends on the coefficient of rand.